Files/UserControls/NavigationToolbar.xaml

```diff
@@ -841,6 +841,8 @@
                 PlaceholderText="Search"
                 QuerySubmitted="SearchRegion_QuerySubmitted"
                 SuggestionChosen="SearchRegion_SuggestionChosen"
+                Background="{ThemeResource SolidBackgroundFillColorQuarternary}"
+                BorderBrush="{ThemeResource SystemBaseMediumLowColor}"
```

Comment:
Should this use the accent color?

Reply:
The background appears transparent and a text is visible below. With this change, the search textbox has the same opaque background as the path textbox. The textbox below uses this background.

Reply:
I was talking about the border color.

Reply:
@cinqmilleans Instead of changing the color, I think we should just close the search box when it loses focus.

Reply:
I will test this idea.

Reply:
In event SearchRegion_LostFocus, there is a rule to only do when the focused element is a popup. Removing the rule fixes the problem. I don't know what this rule is for, but it must be useful. The event is not triggered by the search results popup.

Reply:
@duke7553 Do you have any idea what the rule is being used for?

Reply:
Maybe this was done for the results popup, but the event is not triggered by this popup.

Reply:
It could be it's to prevent losing focus when right clicking on the search box.

Reply:
This is it, I have tested. I will modify the test to only filter this popup.

Reply:
Thanks!

Reply:
It doesn't seem to be a problem to remove the test on the popup. No popup benefits. The search bar menu is filtered by another test.

There is a similar problem in dark mode, but it doesn't seem to have the same cause. The TextBoxStyle does not seem to work and therefore the box has no background. I searched for a long time without finding why.

Reply:
It turns out that the styles for dark mode were recently updated in WinUI to be semi transparent, the best solution would probably be to hardcode the background.